docs/sources/reference/components/loki/loki.source.api.md

```diff
@@ -120,6 +120,12 @@ loki.source.api "loki_push_api" {
 }
 ```
 
+### Technical details
+
+`loki.source.api` filters out all labels that start with `__` like `__tenant_id__`.
+
+If you need to be able to set the tenant id, you should either make sure the `X-Scope-OrgID` header or use the `loki.process` component.
```

Comment:
```suggestion
`loki.source.api` filters out all labels that start with `__`, for example,  `__tenant_id__`.

If you need to be able to set the tenant ID, you must either make sure the `X-Scope-OrgID` header or use the [`loki.process`][loki.process] component.

[loki.process]: ../loki.process/
```

Add a link back to loki.process... 

and `make sure the X-Scope-OrgID header` is what? Available, set to something?  There's a bit missing here.

Reply:
You're right, it's fixed thanks :)